src/components/exit-this-page/index.md

```diff
@@ -0,0 +1,121 @@
+
+### Choosing a different website to load
+
+When activated, the component by default will take the user to the [homepage of BBC Weather](https://www.bbc.co.uk/weather). You can change this if there’s a more appropriate site for your users.
+
+Avoid websites that might show personalised pages (such as frequently visited, last visited, or suggested links), as this content could put the user at risk.
+
+## Consider what to do with user session data
+
+You’ll need to decide if your service will store the user’s session data after they’ve activated the Exit this page component and left your service.
+
+For example, you could do this in 2 steps:
+1. replace the default website link to a URL in your service, that when visited, will clear the user's session data
+2. redirect the user to another website, such as BBC Weather
+
+## Adding the secondary link
+
+Add the code for the 'secondary link' into the layout file of your service. The 'secondary link' is an additional `govuk-skip-link` that works in a similar way. Place it under the page template's default skip link at the very top of the body.
+
+When the user interacts with the 'secondary link', it will perform the same action as pressing the button to activate Exit this Page.
+
+{{ example({group: "components", item: "exit-this-page", example: "secondary-link", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+The `href` for the 'secondary link' should be the same as the URL used for the button.
+
+The component will still work if the 'secondary link' is not added, but it'll be less accessible overall.
+
+## Enhancement features
+
+These features are included in the component and should work automatically, as long as the user has enabled Javascript their browser. You do not need to do anything to make these work.
+
+### Loading overlay
+
+The 'loading overlay' will appear immediately when the user activates Exit this page.
+
+The overlay helps users with slow connections clear their screen until the browser loads the next page.
+
+### Showing progress for 'Shift key' presses
+
+To show the user the number of times they’ve pressed the shift button, there’s a series of 3 progress dots under the button, provided as a visual enhancement. Each time the shift key is pressed, one of the dots will be filled.
+
+The user has 5 seconds to press the shift key 3 times. After that, the sequence and progress dots will reset.
+
+## Accessibility features and considerations
+
+In addition to the 'Exit this page' button itself, users have 2 other options to activate the component. We've tested and iterated both of these options to be accessible, reliable and as discreet as possible for users.
+
+The keyboard option, where the user can press shift 3 times, gives users a discreet way to activate the component, particularly keyboard-only users. We considered options that used a single key press (such as the 'Esc' key), but we could not be confident that this would not interfere with other user actions.
```

Comment:
```suggestion
The keyboard option, where the user can press shift 3 times, gives users a discreet way to activate the component, particularly keyboard-only users. We considered options that used a single key press (such as the 'esc' key), but we could not be confident that this would not interfere with other user actions.
```